src/main/java/com/powsybl/integrationtest/securityanalysis/jsonconfig/SecurityAnalysisTestCaseJson.java

```diff
@@ -29,18 +29,17 @@ public class SecurityAnalysisTestCaseJson {
     private final String expectedResults;
     private final String inputStateMonitors;
 
-    public SecurityAnalysisTestCaseJson(String id, String inputNetwork, String inputParameters,
-                                        String inputContingencies, String inputStateMonitors, String expectedNetwork,
-                                        String expectedResults) {
-        this.id = id;
+    public SecurityAnalysisTestCaseJson(String id, String inputNetwork, String inputParameters) {
+        this.id = Objects.requireNonNull(id);
         this.inputNetwork = Objects.requireNonNull(inputNetwork);
         this.inputParameters = Objects.requireNonNull(inputParameters);
-        this.inputContingencies = Objects.requireNonNull(inputContingencies);
-        this.inputStateMonitors = inputStateMonitors;
-        this.expectedNetwork = Objects.requireNonNull(expectedNetwork);
-        this.expectedResults = Objects.requireNonNull(expectedResults);
+        this.inputContingencies = "contingencies/" + id + ".json";
```

Comment:
What are you trying to achieve here? I fear this is going to make the tester somehow more complicated to use.
Did you update the documentation for these default values?

Reply:
This was an attempt to make the tester easier to use, especially when adding a new test. It is based on:
- the fact that the field "id" is the test case's export name,
- that the folders will always have the same path, unless we change the project's structure
- the files' extension will not change

Another option, instead of "hard-writing" the paths and the extensions in this method, would be to write them only once in the test plan.